src/java/org/apache/cassandra/io/sstable/format/trieindex/TrieIndexSSTableReader.java

```diff
@@ -295,6 +296,7 @@ public PartitionReader reader(FileDataInput file,
      */
     private RowIndexEntry<?> retrieveEntryIfAcceptable(Operator searchOp, PartitionPosition searchKey, long pos, boolean assumeNoMatch) throws IOException
     {
+        // todo should we update bf here?
```

Comment:
Why? This is not used in exact search, bloom filter has no part to play.

Reply:
right